pyintegration/deephaven2/stream/kafka/_utils.py

```diff
@@ -0,0 +1,47 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+""" The private utility module for kafka. """
+import jpy
+
+_JProperties = jpy.get_type("java.util.Properties")
+_JHashMap = jpy.get_type("java.util.HashMap")
+_JHashSet = jpy.get_type("java.util.HashSet")
+_JPythonTools = jpy.get_type("io.deephaven.integrations.python.PythonTools")
+IDENTITY = object()  # Ensure IDENTITY is unique.
+
+
+def _dict_to_j_properties(d):
+    r = _JProperties()
+    for key, value in d.items():
+        if value is None:
+            value = ''
+        r.setProperty(key, value)
+    return r
+
+
+def _dict_to_j_map(d):
```

Comment:
These utilities will be useful for more modules I think.  At least the upcoming stream.cdc, so perhaps a better place for them would be at a higher directory level.